pkgs/development/python-modules/bx-python/default.nix

```diff
@@ -2,38 +2,51 @@
   lib,
   fetchFromGitHub,
   buildPythonPackage,
-  pythonOlder,
   numpy,
   cython,
   zlib,
   python-lzo,
-  nose,
+  pytestCheckHook,
+  setuptools,
+  oldest-supported-numpy,
 }:
 
 buildPythonPackage rec {
   pname = "bx-python";
-  version = "0.11.0";
-  format = "setuptools";
-
-  disabled = pythonOlder "3.8";
+  version = "0.12.0";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "bxlab";
     repo = "bx-python";
     rev = "refs/tags/v${version}";
-    hash = "sha256-evhxh/cCZFSK6EgMu7fC9/ZrPd2S1fZz89ItGYrHQck=";
+    hash = "sha256-ZpZjh7OXdUY7rd692h7VYHzC3qCrDKFme6r+wuG7GP4=";
   };
 
-  nativeBuildInputs = [ cython ];
+  postPatch = ''
+    substituteInPlace pytest.ini \
+      --replace-fail "--doctest-cython" ""
```

Comment:
I’m curious about what’s going on here. Can we not run the Cython doctests? (I mean, I guess we can’t run the tests at all right now, but…)

Reply:
pytest-cython, which provides that option, isn't packaged.

Reply:
Makes sense, thanks! Can you leave a comment to that effect in case anyone wants to try and get the tests working later?

Reply:
Added a comment